package.json

```diff
@@ -38,54 +38,59 @@
     "ng-recaptcha": "^9.0.0",
     "ngx-cookie-service": "^15.0.0",
     "rxjs": "~6.6.0",
-    "swiper": "^7.0.0",
     "tailwind-colors": "^1.1.0",
-    "tslib": "^2.0.0",
-    "zone.js": "0.11.8"
+    "tslib": "^2.3.0",
+    "zone.js": "0.13.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "14.2.7",
-    "@angular-eslint/eslint-plugin": "14.0.4",
-    "@angular-eslint/eslint-plugin-template": "14.0.4",
-    "@angular-eslint/template-parser": "14.0.4",
-    "@angular/cli": "~14.2.0",
-    "@angular/compiler-cli": "14.2.8",
-    "@angular/language-service": "14.2.8",
+    "@angular-devkit/build-angular": "16.0.4",
+    "@angular-devkit/core": "16.0.4",
+    "@angular-devkit/schematics": "16.0.4",
+    "@angular-eslint/eslint-plugin": "16.0.3",
+    "@angular-eslint/eslint-plugin-template": "16.0.3",
+    "@angular-eslint/template-parser": "16.0.3",
+    "@angular/cli": "~16.0.0",
+    "@angular/compiler-cli": "16.0.4",
+    "@angular/language-service": "16.0.4",
     "@gammastream/scully-plugin-http404": "^1.0.4",
     "@gammastream/scully-plugin-sitemap": "^1.0.7",
-    "@nrwl/cli": "15.0.3",
-    "@nrwl/cypress": "15.0.3",
-    "@nrwl/eslint-plugin-nx": "15.0.3",
-    "@nrwl/jest": "15.0.3",
-    "@nrwl/linter": "15.0.3",
-    "@nrwl/node": "^15.9.3",
-    "@nrwl/nx-cloud": "15.0.0",
-    "@nrwl/workspace": "15.0.3",
+    "@nx/cypress": "16.3.1",
+    "@nx/eslint-plugin": "16.3.1",
+    "@nx/jest": "16.3.1",
+    "@nx/js": "16.3.1",
+    "@nx/linter": "16.3.1",
+    "@nx/node": "16.3.1",
+    "@nx/workspace": "16.3.1",
+    "@schematics/angular": "16.0.4",
     "@types/fs-extra": "9.0.7",
-    "@types/jest": "28.1.8",
+    "@types/jest": "29.4.4",
     "@types/node": "18.7.1",
-    "@typescript-eslint/eslint-plugin": "5.41.0",
-    "@typescript-eslint/parser": "5.41.0",
+    "@typescript-eslint/eslint-plugin": "5.59.8",
+    "@typescript-eslint/parser": "5.59.8",
     "cypress": "^8.3.0",
     "eslint": "8.15.0",
     "eslint-config-prettier": "8.1.0",
     "eslint-plugin-cypress": "^2.10.3",
     "fast-xml-parser": "^4.0.8",
     "fs-extra": "^9.1.0",
-    "jest": "28.1.3",
-    "jest-environment-jsdom": "28.1.1",
-    "jest-preset-angular": "12.2.2",
-    "ng-packagr": "14.2.2",
-    "nx": "15.0.3",
+    "jest": "29.4.3",
+    "jest-environment-jsdom": "29.4.3",
+    "jest-preset-angular": "13.1.1",
+    "ng-packagr": "16.0.1",
+    "nx": "16.3.1",
+    "nx-cloud": "16.0.5",
+    "patch-package": "^7.0.0",
     "postcss": "^8.3.9",
     "postcss-import": "14.1.0",
     "postcss-preset-env": "7.5.0",
     "postcss-url": "10.1.3",
+    "postinstall-postinstall": "^2.1.0",
```

Comment:
was this added by migrations?

Reply:
no, this was added manually for patch-package

Reply:
no, this was added manually for patch-package

Reply:
ah, that's because of yarn v1, got it